T/TetGen/build_tarballs.jl

```diff
@@ -57,7 +60,7 @@ install_license LICENSE
 """
 
 platforms = supported_platforms(; experimental=true)
-# platforms=[Platform("x86_64", "linux"; libc="glibc")]
+#platforms=[Platform("x86_64", "linux"; libc="glibc")]
```

Comment:
What's the point of this line?  Can we just remove it?
```suggestion
```

Reply:
It is for my own testing with TetGen.jl - just as  a note what to do during test cycles in combination with the wrapper package. For this purpose I upload into my own repo for the jll before submitting here.
Can put this elsewhere and remove this.

Reply:
You can pass the platform you want to build for on the command line: https://docs.binarybuilder.org/stable/#Manually-create-or-edit-build_tarballs.jl  That's what I always do, without touching the script